agent/mailTask.py

```diff
@@ -197,19 +199,15 @@ def prepare_job(self, msg):
         """
         self.prepare_try_build(msg.revision)
 
-        cmd = [sys.executable,
-               "runtest.py",
-               "re",
-               "test.suite",
-               "--max-run=" + str(msg.max_run),
-               "--max-retry=" + str(msg.max_retry)]
-        if msg.calc_si != "":
-            cmd.append(msg.calc_si)
-        if msg.advance != "":
-            cmd.append(msg.advance)
-
-        logger.debug("cmd: %s" % ' '.join(cmd))
-        subprocess.call(cmd, env=os.environ.copy())
+        self.kwargs['INDEX_CONFIG_NAME'] = 'runtimeDctGenerator.json'
```

Comment:
We do not always use 'runtimeDctGenerator.json'. Is it possible to do different json files?